Optimizing the Security of your Skywallet.md

```diff
@@ -0,0 +1,48 @@
+# Optimizing the Security of your Skywallet
+
+Even though the Skywallet provides greater security when compared to a Desktop Wallet, it is essential to follow these security optimizations to get maximum security benefit from your Skywallet.
+
+Two things are necessary for a third party to access your funds in Skywallet, which is either the PIN code of the Skywallet or the Seed of the Skywallet.
+
+You are recommended to create a secure PIN code, although, unless your Skywallet is lost or got stolen, no one can access it using the PIN code.
+
+However, this is not the case for a Seed, and any hacker/malware which gets access to your Seed of the Skywallet can remotely access or steal your funds.
+
+So when it comes to security optimizations, these are the two vital components to be considered.
+
+## PIN Code
+
+Your PIN code is a four-digit combination, and a brand new Skywallet which you receive via shipment does not have a PIN code.
+
+Hence, it is essential first to set a PIN code right after you receive your Skywallet. If you want to know more about how to set the PIN code and personalize your Skywallet, please visit the user manual [Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet).
+
+Though the PIN code is a four-digit combination, it would not be easy for an attacker/hacker to try the different combinations and work it out. 
+
+Because after each failed attempt, one has to wait for a specific time before attempting the next combination and with each failed attempt the waiting time increases exponentially, making it impossible for an attacker/hacker to obtain the PIN code that way.
```

Comment:
, it is very unlikely that an attacker can gain access to  your device this way.